domains/transfer/Transfer.API/API/Cvr/Api/v2024_01_03/Controllers/CvrController.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using API.Cvr.Api.Clients.Cvr;
+using API.Cvr.Api.Models;
+using API.Cvr.Api.v2024_01_03.Dto.Responses;
+using Asp.Versioning;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace API.Cvr.Api.v2024_01_03.Controllers;
+
+[Authorize]
+[ApiController]
+[ApiVersion(20240103)]
+[Route("api/cvr")]
+public class CvrController(CvrClient client) : Controller
+{
+    /// <summary>
+    /// Get CVR registered company information for multiple CVR numbers
+    /// </summary>
+    /// <response code="200">Successful operation</response>
+    /// <response code="404">CVR company not found</response>
+    [ProducesResponseType(typeof(CvrCompanyListResponse), 200)]
+    [ProducesResponseType(typeof(void), 404)]
+    [HttpPost]
+    public async Task<ActionResult<CvrCompanyListResponse>> GetCvrCompanies([FromBody] List<string> cvrNumbers)
+    {
+        var parsedCvrNumbers = cvrNumbers.Select(CvrNumber.TryParse).OfType<CvrNumber>().ToList();
+
+        if (!parsedCvrNumbers.Any())
+            return Ok(new CvrCompanyListResponse(new List<CvrCompanyDto>()));
+
+        var searchResult = await client.CvrNumberSearch(parsedCvrNumbers);
+        if (searchResult?.hits?.hits == null)
+            return Ok(new CvrCompanyListResponse(new List<CvrCompanyDto>()));
+
+        var companies = searchResult.hits.hits
+            .Select(hit => ToDto(hit._source?.Vrvirksomhed))
+            .Where(dto => dto != null)
+            .ToList();
```

Comment:
Put this in a ToDto function

Reply:
ToDto function, er kun beregnet til 1x dto, denne logik omhandler at putte flere dto'er i en liste. Ved ikke om toDto er det rigtige sted den skal være. en separat metode så?

Reply:
Du kan vel bare have flere ToDto metoder, der har forskellige input parametre?

Reply:
Done:

https://github.com/Energinet-DataHub/energy-origin/pull/558/commits/ade92237814c50ef674632ffd110dbbdbed85b75